src/DIRAC/DataManagementSystem/Client/DataManager.py

```diff
@@ -432,6 +434,9 @@ def putAndRegister(self, lfn, fileName, diracSE, guid=None, path=None, checksum=
         'overwrite' removes file from the file catalogue and SE before attempting upload
         """
 
+        if len(os.path.basename(lfn)) >= MAX_FILENAME_LENGTH:
```

Comment:
```suggestion
        if len(os.path.basename(lfn)) > MAX_FILENAME_LENGTH:
```
We need `>` so we can use all the 128 characters, and not only 127, right?

(Sorry for dripping comments in like this.)

Reply:
I actually did it on purpose because I seem to remember being bitten by something like it and the field being truncated, but I really can't recall what it was :laughing:

Reply:
Adding a comment, and changing the message to read 127 max characters is also fine.

Reply:
I went with the approach of `>` and watch the world burn :laughing: 
Since my memory is not so good, maybe I am mistaken, or it will be a reminder of why I had that n my mind :-p